pcdsdevices/state.py

```diff
@@ -275,6 +275,15 @@ def __init__(self, *, name, **kwargs):
             self._sub_signals.append(sig)
             self._sub_map[signal_name] = sig
 
+    def describe(self):
+        # Base description information
+        sub_sigs = [sig.name for sig in self._sub_signals]
+        desc = {'source': 'SUM:{}'.format(','.join(sub_sigs)),
```

Comment:
Not sure why this is called `SUM` but I don't have any reasonable suggestions for it.

Reply:
Good question. Meant it as short for summary.  Open to suggestions.